tst/testinstall/kernel/records.tst

```diff
@@ -28,6 +28,8 @@ Error, NameRName: <rnam> must be a positive small integer (not the integer -1)
 gap> NameRNam(fail);
 Error, NameRName: <rnam> must be a positive small integer (not the value 'fail\
 ')
+gap> NameRNam(2^29);
+Error, NameRName: <rnam> must be a valid rnam (not the integer 536870912)
```

Comment:
This error message (and several before it) still has the old, incorrect `NameRName` in it, which is why the Travis tests failed.

Reply:
sorry, been away from my computer. Now hopefully fixed and poked.